### DISCUSSION TITLE
can we get the real domain name/ hostname of shopify  stores.

### DISCUSSION THREAD

**Discussion by sudhir-pratap-singh** [osiset/laravel-shopify#1237] [General]
for example => i want to fetch www.example.com instead of 'example.myshopfy.com' during app install. 
Is this possible??

**Comment by d-shannon** [comment]
You could do something like this in your AfterAuthenticateJob:

```
$response = $this->api->rest('GET', '/admin/shop.json')['body'];

$this->shop->domain = data_get($response, 'shop.domain');
$this->shop->save();
```

**Reply by sudhir-pratap-singh** [reply]
thanks buddy, great suggestion

**Comment by sudhir-pratap-singh** [comment]
by default customer email is saving as shop@domain->name , but this is not the correct url for store owners. How to find that proper email of store owner?

**Reply by d-shannon** [reply]
Open /admin/shop.json directly in your web browser. You will see everything you need there.
